.github/workflows/flux-update-scheduled-check.yml

```diff
@@ -0,0 +1,28 @@
+name: Flux Update Schedule Check
+on:
+  schedule:
+    - cron:  '0 12 * * SUN'
+
+jobs:
+  flex-schedule-check:
+    name: Flux Update Schedule Check
+    runs-on:
+      - self-hosted
+      - small
+    steps:
```

Comment:
lets limit this to only run on `main`, once we create release branches we don't need/want this job running on those branches.